### ISSUE TITLE
Crash when switching to Advanced Mode

### ISSUE THREAD

**Issue by cachitas** [kristinbranson/JAABA#317]
Using JAABA 0.6.0 from master branch.

```
Warning: Unknown parameter name: panel_similar_ResizeFcn, ignoring
 
> In myparse (line 26)
  In JLabel_JLabel_OpeningFcn (line 64)
  In gui_mainfcn (line 220)
  In JLabel (line 37)
  In @(hObject,eventdata)JLabel('panel_similar_ResizeFcn',hObject,eventdata,guidata(hObject))
  In JLabel>updatePanelPositions (line 4599)
  In JLabel>UpdateGUIToMatchAdvancedMode (line 7013)
  In JLabel>menu_edit_advanced_mode_Callback (line 8061)
  In gui_mainfcn (line 95)
  In JLabel (line 37)
  In @(hObject,eventdata)JLabel('menu_edit_advanced_mode_Callback',hObject,eventdata,guidata(hObject)) 
Error using myparse (line 20)
Input 3 is not a string.

Error in JLabel_JLabel_OpeningFcn (line 64)
[defaultPath,...

Error in gui_mainfcn (line 220)
    feval(gui_State.gui_OpeningFcn, gui_hFigure, [], guidata(gui_hFigure), varargin{:});

Error in JLabel (line 37)
    gui_mainfcn(gui_State, varargin{:});

Error in @(hObject,eventdata)JLabel('panel_similar_ResizeFcn',hObject,eventdata,guidata(hObject))

 
Error using JLabel>updatePanelPositions (line 4599)
Error while evaluating Panel SizeChangedFcn

Warning: Unknown parameter name: panel_similar_ResizeFcn, ignoring
 
> In myparse (line 26)
  In JLabel_JLabel_OpeningFcn (line 64)
  In gui_mainfcn (line 220)
  In JLabel (line 37)
  In @(hObject,eventdata)JLabel('panel_similar_ResizeFcn',hObject,eventdata,guidata(hObject)) 
Error using myparse (line 20)
Input 3 is not a string.

Error in JLabel_JLabel_OpeningFcn (line 64)
[defaultPath,...

Error in gui_mainfcn (line 220)
    feval(gui_State.gui_OpeningFcn, gui_hFigure, [], guidata(gui_hFigure), varargin{:});

Error in JLabel (line 37)
    gui_mainfcn(gui_State, varargin{:});

Error in @(hObject,eventdata)JLabel('panel_similar_ResizeFcn',hObject,eventdata,guidata(hObject))

 
Error while evaluating DestroyedObject SizeChangedFcn
```

**Comment by mkabra** [comment]
Hi Hugo,

What version of matlab are you using?

**Comment by cachitas** [comment]
Tested on 2013b and 2015a.

**Comment by mkabra** [comment]
Hi Hugo,

I've fixed the bug. Switching to advanced mode should not fail.

Mayank

**Comment by cachitas** [comment]
Thanks! 
Commit with the fix: 0f191d7e7a971cac5a261368c082b43376d4d4f9.
